src/components/layout/CardHeader.tsx

```diff
@@ -10,7 +10,7 @@ import CardTitle from './CardTitle';
 
 type ComponentProps = {
   title?: string;
-
+  size?: 'sm' | 'md' | 'lg';
```

Comment:
Not sure if I'm missing something, but I don't see the `sm` and `md` values being used differently?

Reply:
By looking at how this is used, perhaps it makes more sense to pass `classes="px-6"` explicitly from some parent component when some other prop is provided there, instead of defining a new prop here.